terraform/outputs.tf

```diff
@@ -0,0 +1,26 @@
+output "vpc_id" {
+  value = "${module.vpc.id}"
+}
+
+output "logs_bucket_id" {
+  value = "${aws_s3_bucket.logs.id}"
```

Comment:
Just to be clear, outputs are not necessary for top level resources like these, but you're doing it here for consistency?